index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- fontawesomecdn -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <!-- link to css file -->
+    <link rel="stylesheet" href="./style.css">
+    <title>Lynette || My Portfolio</title>
+</head>
+<body>
+    <header>
+
+        
+        <div class="container">
+       <div class="wrapper">
+                <nav>
+                    <a href="#" class="welcome">Welcome</a>
+                    <a class="burger"><i class="fas fa-bars"></i></a>
+                    <ul class="nav-list">
+                        <li><a class="nav-item" href="#">Portfolio</a></li>
+                        <li><a class="nav-item" href="#">About</a></li>
+                        <li><a class="nav-item" href="#">Contact</a></li>
+                    </ul>
```

Comment:
- [ ] [OPTIONAL] It's good you added these nav links but they aren't needed for this task. It will a good thing to keep them out for now. Let's try to follow the project requirement and not go beyond what was asked. At Microverse students learn to build projects sequentially.